packages/core/src/icon/index.ts

```diff
@@ -6,6 +6,7 @@
 
 export * from './icon.element.js';
 export * from './icon.service.js';
+export { renderIcon } from './icon.renderer.js';
```

Comment:
Why is this change needed?

Reply:
Sorry, undesired commits came along, fixed it.